.github/workflows/main.yaml

```diff
@@ -0,0 +1,21 @@
+name: Main
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+env:
+  CARGO_TERM_COLOR: always
```

Comment:
Does it work on the GH Actions Log? Won't it display the colors codes instead of coloring the output?

Reply:
It's required, actually